pkg/config/generate_test.go

```diff
@@ -66,8 +67,13 @@ func testRequiredFields(cs *api.OpenShiftManagedCluster, t *testing.T) {
 	assert(c.Images.LogBridge != "", "logbridge image")
 
 	assert(c.ServiceAccountKey != nil, "service account key")
-	assert(len(c.HtPasswd) != 0, "htpassword")
-	assert(len(c.AdminPasswd) != 0, "admin password")
+
+	if pc.TestConfig.RunningUnderTest {
```

Comment:
@kwoodson I think you should hard-set `pc.TestConfig.RunningUnderTest` above in TestGenerate and remove the if here.